content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,246 @@
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 3m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 5m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 7m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 9m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 11m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 13m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 15m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 17m10s elapsed]
+hpegl_caas_cluster.test: Still modifying... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a, 19m10s elapsed]
+hpegl_caas_cluster.test: Modifications complete after 19m18s [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a]
+
+Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
+```
+
+From HPE GreenLake edge-to-cloud platform, launch the HPE GreenLake Central console. Navigate to **Dashboard -> Manage your Private Cloud -> Containers** and select the **tf-test** cluster created. You will see additional nodes with **Node Pool** name as "**test-node-pool**" being created successfully.
+
+![](/img/cluster_detail_page.jpg)
+
+# Scale Options
+
+The above example is specifically for adding a single worker node pool to an existing cluster. Below are all the possible options available for scaling.
+
+1. Add worker node pools: We can add multiple node pools by simply declaring corresponding **worker_nodes** in the same cluster resource.
+2. Reduce worker node pools: Remove **worker_nodes** associated with a specific node pool from the cluster resource
+3. Increase/decrease worker node count: Updating the **count** field increases or decreases the number of nodes under each node pool. 
+4. Increase/decrease default worker node count: Every cluster by default has a worker node with the node pool name “**worker**” even if **worker_nodes** are not declared in the Terraform configuration. This originally comes from what's declared in the cluster blueprint. We can override and update the count and machine blueprint for this default worker by declaring **worker_nodes** with the name “**worker**”. 
+
+Note: If we remove the default node pool (**worker_nodes** with name “**worker**”), the default configuration coming from the cluster blueprint shall be retained.
+
+# Summary
+
+In this blog, we covered how to scale Kubernetes clusters with Terraform provider for HPE GreenLake. We showed you how to update an existing cluster with additional worker nodes. We also discussed several options available to increase or reduce worker nodes across different node pools for Kubernetes clusters.
```

Comment:
Change "we" to "I" in each instance in the paragraph below so it reads:
In this blog, I covered how to scale...
I showed you how to...
I also discussed several options...